packages/nat/package.json

```diff
@@ -10,9 +10,11 @@
   },
   "scripts": {
     "build": "exit 0",
+    "build:types": "tsc --build tsconfig.build.json",
+    "clean:types": "git clean -f '*.d.ts*'",
     "test": "ava test/**/test-*.js",
     "lint-fix": "eslint --fix",
-    "lint-check": "eslint"
+    "lint": "tsc"
```

Comment:
Do we lose eslint here?

Reply:
It wasn't working. It wasn't erroring in CI because the script wasn't called "lint"

Reply:
Cool story, tracking https://github.com/endojs/endo/issues/2134